src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -42,196 +42,197 @@
 
 @Ignore
 @TestPropertySource(properties = { "notification.ttl.value:1", "aggregated.collection.ttlValue:1",
-		"notification.failAttempt:1" })
+        "notification.failAttempt:1" })
 @ContextConfiguration(initializers = TestContextInitializer.class)
 public class TestTTLSteps extends FunctionalTestBase {
-	private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
-	private static final String BASE_URL = "localhost";
-	private static final String ENDPOINT = "/missed_notification";
-	private static final String SUBSCRIPTION_NAME = "Subscription_1";
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
+    private static final String BASE_URL = "localhost";
+    private static final String ENDPOINT = "/missed_notification";
+    private static final String SUBSCRIPTION_NAME = "Subscription_1";
 
```

Comment:
Which code parts has changed? everything is green marked.

Reply:
You can remove tab changes by clicking Diff settings-> Hide whitespace changes